docs/release-notes/dictionary-releases.md

```diff
@@ -3,14 +3,29 @@ id: dictionary-releases
 title: Dictionary Releases
 ---
 
-A DCC dictionary release happens when there has been an update to the clinical dataset in the **[ARGO Data Dictionary](/dictionary)**. Dictionary releases happen whenever needed. 
+A DCC dictionary release happens when there has been an update to the clinical dataset in the **[ARGO Data Dictionary](/dictionary)**. Dictionary releases happen whenever needed.
 
-## Dictionary Release 1.0 
+<!---
+## Dictionary Release
 
 **Release Date:**
 
-### New Updates 
-Coming soon.
+### New Updates
 
-### Bug Fixes 
 Coming soon.
+
+### Bug Fixes
+
+None to report.
+------>
+
+## Initial Dictionary Release 1.0.0
```

Comment:
should the version number match the number in here? https://5eecf683dc248e000761dd83--argo-docs.netlify.app/dictionary

Reply:
this dictionary is not yet updated.  waiting on pr from hardeep to be ready. it will match later.